Base/QTGUI/qSlicerApplication.cxx

```diff
@@ -325,6 +354,17 @@ qSlicerApplication::qSlicerApplication(int &_argc, char **_argv)
 //-----------------------------------------------------------------------------
 qSlicerApplication::~qSlicerApplication()
 {
+  // We have to delete PythonConsole if it is not deleted already
+  // and not owned by a widget (it is owned and deleted by a widget if it is added
+  // to the GUI).
```

Comment:
* I think this should be protected by `#ifdef Slicer_USE_PYTHONQT ` and `qSlicerCoreApplication::testAttribute(qSlicerCoreApplication::AA_DisablePython)`

* Fix indent